specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -260,6 +260,9 @@ model AcsEmailDeliveryReportReceivedEventData {
   /** The recipient Email Address */
   recipient?: string;
 
+  /** The Internet Message Id of the email been sent */
+  internetMessageId?: string;
```

Comment:
Should this be a required property?

Reply:
There's a delivery report with status value ="Suppressed", for those events we don't have the internetMessageId property, so this should be optional.